content/_partials/downloadlist.html.haml

```diff
@@ -113,3 +113,51 @@
         %a.item{:href=>'/changelog'} Changelog
         |
         %a.item{:href=>'http://mirrors.jenkins-ci.org/war/'} Past Releases
+  
+    .col-md-6.text-xs-center
+      %h3
+        Preview
+
+      %p
+        This is 2.0 beta
+
+      %p
+        .btn-group{:style=>'margin-top: 1.3125rem;'}
+          %a.btn.btn-primary.chromeOnly{:href=>'http://mirrors.jenkins-ci.org/war-rc/2.0/jenkins.war'}
+            %i.icon-box-add
+            = site.jenkins.latest
```

Comment:
We don't have a variable for the beta version, but reusing this one is certainly wrong.